packages/react/src/themes/teams/components/Checkbox/checkboxVariables.ts

```diff
@@ -1,22 +1,75 @@
-import { CheckboxVariables as BaseCheckboxVariables } from '../../../base/components/Checkbox/checkboxVariables'
+import { pxToRem } from '../../../../lib'
 
-export type CheckboxVariables = Partial<BaseCheckboxVariables>
+export interface CheckboxVariables {
+  rootPadding: string
+
+  checkboxTextColor: string
+  checkboxBackground: string
+  checkboxBorderColor: string
+  checkboxBorderStyle: string
+  checkboxBorderRadius: string
+  checkboxBorderWidth: string
+  checkboxIndicatorColor: string
+  checkboxGap: string
+  checkboxMargin: string
+  checkboxPadding: string
+
+  checkboxTextColorHover: string
+  checkboxBorderColorHover: string
+  checkboxCheckedBackgroundHover: string
+  checkboxCheckedTextColor: string
+  checkboxCheckedBackground: string
+  checkboxCheckedBorderColor: string
+  checkboxCheckedIndicatorColor: string
+
+  toggleBorderRadius: string
+  toggleIndicatorColor: string
+  toggleMargin: string
+  togglePadding: string
+  toggleCheckedPadding: string
+
+  disabledColor: string
+  disabledCheckboxBackground: string
+  disabledCheckboxBackgroundChecked: string
+  disabledCheckboxBorderColor: string
+  disabledCheckboxCheckedIndicatorColor: string
+  disabledToggleIndicatorColor: string
+}
+
+const [toggleMovementDistance, checkboxPadding] = [20, 2].map(v => pxToRem(v))
```

Comment:
Can you please just inline this because it's hard to read things like this...

```
togglePadding: `${pxToRem(2)} ${pxToRem(20)} ${pxToRem(2)} ${pxToRem(2)}`,
```

Reply:
I disagree in this case, those variables are important for maintaining proper behavior of the toggle (see the togglePadding and toggleCheckedPadding variables' values, they both use the movement distance. An aesthetic problem could occur if the inline values are changed in one and not the other.

Reply:
```
const [toggleMovementDistance, padding] = [20, 2].map(v => pxToRem(v))
```
...
```
togglePadding: `${padding} ${toggleMovementDistance} ${padding} ${padding}`,
toggleCheckedPadding: `${padding} ${padding} ${padding} ${toggleMovementDistance}`,
```

Reply:
The `map` syntax looks strange to my eyes, but let's not bikeshed too long on it.  Progress trumps this one in priority,

Reply:
I changed it to not use map, just two separate variables.